Maddori.Apple/Maddori.Apple/Screen/MyBox/EditFeedbackFromMe/EditFeedbackFromMeViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  EditFeedbackFromMeViewController.swift
+//  Maddori.Apple
+//
+//  Created by 김유나 on 2022/11/06.
+//
+
+import UIKit
+
+import SnapKit
+
+final class EditFeedbackFromMeViewController: AddFeedbackContentViewController {
+    
+    private let model = FeedbackFromMeModel.mockData
+    private var isFeedbackTypeChanged: Bool = false {
+        didSet {
+            if !(isTextInputChanged() || isFeedbackTypeChanged) {
+                feedbackDoneButton.isDisabled = true
+            } else {
+                feedbackDoneButton.isDisabled = false
+            }
+        }
+    }
+    
+    // MARK: - life cycle
+    
+    override func configUI() {
+        super.configUI()
+        setupFeedbackType()
+        setupFeedbackKeyword()
+        setupFeedbackContent()
+        setupFeedbackStart()
+        hideEditFeedbackUntilLabel()
+        detectChangeOfFeedbackType()
+    }
+    
+    // MARK: - func
+    
+    private func setupFeedbackType() {
+        switch model.feedbackType {
+        case .continueType:
+            self.feedbackTypeButtonView.touchUpToSelectType(.continueType)
+        case .stopType:
+            self.feedbackTypeButtonView.touchUpToSelectType(.stopType)
+        }
+    }
+    
+    private func setupFeedbackKeyword() {
+        feedbackKeywordTextField.text = model.keyword
+        setCounter(count: model.keyword.count)
+    }
+    
+    private func setupFeedbackContent() {
+        feedbackContentTextView.text = model.info
+        feedbackContentTextView.textColor = .black100
+    }
+    
+    private func setupFeedbackStart() {
+        if let start = model.start {
+            feedbackStartSwitch.isOn = true
+            feedbackStartTextViewLabel.isHidden = false
+            feedbackStartTextView.isHidden = false
+            feedbackStartTextView.text = start
+            feedbackStartTextView.textColor = .black100
+        }
+    }
+    
+    private func hideEditFeedbackUntilLabel() {
+        editFeedbackUntilLabel.isHidden = true
+        feedbackDoneButtonView.snp.remakeConstraints {
+            $0.bottom.equalToSuperview()
+            $0.leading.trailing.equalToSuperview()
+            $0.height.equalTo(95)
+        }
+    }
+    
+    @objc override func willHideKeyboard(notification: NSNotification) {
```

Comment:
p5;

```suggestion
    // MARK: selector
    
    @objc override func willHideKeyboard(notification: NSNotification) {
```

붙여줄 수 있나요??

Reply:
[ee57e0e](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/83/commits/ee57e0e448040874197b815c5bfc361d77dd7292)
어머머 이걸 빼먹다니,,,, 감사합니당 !!! 👍